extensions/wikia/Wall/WallMessage.class.php

```diff
@@ -546,14 +546,25 @@ public function getMainPageText() {
 		return $wallOwnerName;
 	}
 
+	private function getTitleBaseText( Title $title ): string {
+		$titleText = $title->getText();
+		$slashPosition = strpos( $titleText, '/' );
+
+		if ( $slashPosition === false ) {
+			return $titleText;
+		}
+
+		return substr( $titleText, 0, $slashPosition );
+	}
+
 	public function getWallOwner() {
 		if ( $this->title->inNamespace( NS_USER_WALL_MESSAGE ) ) {
-			$ownerUserName = $this->title->getBaseText();
+			$ownerUserName = $this->getTitleBaseText( $this->title );
 			return User::newFromName( $ownerUserName, false );
 		}
 
 		// horribile visu: Forum threads treat board name as user!
```

Comment:
Shouldn't this be fixed?

Reply:
No.

Reply:
But the comment shouldn't be removed either?

Reply:
It helps explain the situation to other developers who have not worked with this code before.